mailscanner/functions.php

```diff
@@ -244,29 +249,29 @@ function html_start($title, $refresh = 0, $cacheable = true, $report = false)
     echo '<tr>' . "\n";
     echo '<td valign="bottom" align="left" class="jump">' . "\n";
     echo '<form action="./detail.php">' . "\n";
-    echo '<p>Jump to message:<input type="text" name="id" value="' . $message_id . '"></p>' . "\n";
+    echo '<p>'.$lang['jumpmessage03'].'<input type="text" name="id" value="' . $message_id . '"></p>' . "\n";
     echo '</form>' . "\n";
     echo '</table>' . "\n";
     echo '<table cellspacing="1" class="mail">' . "\n";
-    echo '<tr><td class="heading" align="center">Current User</td><td class="heading" align="center">Current Sytem Time</td></tr>' . "\n";
+    echo '<tr><td class="heading" align="center">'.$lang['cuser03'] . '</td><td class="heading" align="center">' . $lang['cst03'] . '</td></tr>' . "\n";
     echo '<tr><td>' . $_SESSION['fullname'] . '</td><td><span id="clock">&nbsp;</span></td></tr>' . "\n";
     echo '</table>' . "\n";
     echo '</td>' . "\n";
 
     echo '<td align="left" valign="top">' . "\n";
     echo '   <table border="0" cellpadding="1" cellspacing="1" class="mail">' . "\n";
-    echo '    <tr> <th colspan="2">Color Codes</th> </tr>' . "\n";
-    echo '    <tr> <td>Bad Content/Infected</TD> <td class="infected"></TD> </TR>' . "\n";
+    echo '    <tr> <th colspan="2">' . $lang['colorcodes03'] . '</th> </tr>' . "\n";
+    echo '    <tr> <td>' . $lang['ccodes01_03'] . '</TD> <td class="infected"></TD> </TR>' . "\n";
```

Comment:
This variable name is not self explanatory, should be `$lang['badcontentinfected_03']`